node/src/Commands.ts

```diff
@@ -2360,20 +2372,22 @@ function addReadOptions(options: StreamReadOptions, args: string[]) {
         args.push("BLOCK");
         args.push(options.block.toString());
     }
-}
 
-function addStreamsArgs(keys_and_ids: Record<string, string>, args: string[]) {
-    args.push("STREAMS");
+    return args;
+}
 
-    const pairs = Object.entries(keys_and_ids);
+function addStreamsArgs(keys_and_ids: Record<string, string>): string[] {
+    const args = ["STREAMS"];
 
-    for (const [key] of pairs) {
+    for (const key of Object.keys(keys_and_ids)) {
         args.push(key);
```

Comment:
doesn't it work to just do `args.push(Object.keys(keys_and_ids))` or `args.push(...Object.keys(keys_and_ids))`